### DISCUSSION TITLE
"dynamic notifications" allows only 1 notification

### DISCUSSION THREAD

**Discussion by krizzzly** [solspace/craft-express-forms#101] [Questions & Answers] [answered]
Express Forms has Admin-Notifications and Submitter-Notifications.
But "Dynamic-Notifications" in form options seems to allow only 1 notification.
Is there something i am missing here. Thx for clarification.

Kind regards

Chris

PS: For automated form testing i would like to be able to overwrite the Admin-Notifications and Submitter-Notifications from GUI, and add dynamic Notifications to "testing-mailboxes".

**Comment by kjmartens** [comment]
I haven't tested this myself, but I believe you could have multiple dynamic recipients by specifying multiple iterations here:

```
{{ form.openTag({
  dynamicRecipients: {
    myFieldHandle: {
      map: {
        sales: "sales@example.com",
        service: "service@example.com",
        support: ["support@example.com", "another@example.com"]
      },
      template: "template-filename.twig",
    },
    myFieldOtherHandle: {
      map: {
        a: "a@example.com",
        b: "b@example.com",
        c: ["c@example.com", "another@example.com"]
      },
      template: "template-filename.twig",
    },
  }
}) }}
```

**Comment by krizzzly** [comment]
I thought actually about "dynamic notifications" not "dynamic recipients".
https://craft.express/forms/v1/overview/email-notifications.html#dynamic-template

The goal is: the "Admin notification" setting from GUI is overwritten bei config, "Submitter notification" is send out as set in GUI( or it is also overwritten by config.)

**Comment by kjmartens** [comment, marked as answer]
Sorry for the misunderstanding! Yes, that is correct that it can only be set up once for a form, not multiple times. One potential solution would be to have no Admin notification configured and use template conditionals to handle values inside the Dynamic Notifications parameter approach, depending of course, on how this is to be handled. 🙂
